services/auth/oauth2.go

```diff
@@ -103,6 +103,7 @@ func (o *OAuth2) userIDFromToken(ctx context.Context, tokenSHA string, store Dat
 
 				store.GetData()["IsActionsToken"] = true
 				store.GetData()["ActionsTaskID"] = task.ID
+				store.GetData()["ActionsTask"] = task
```

Comment:
We can load the task from database when we need it but not store it in context?

Reply:
I honestly don't remember why I did it this way. It's been a while. Let me take another look.